### ISSUE TITLE
[Request] Action Bar Icon Aspect Ratio

### ISSUE THREAD

**Issue by Zelfie** [ElvUI-WotLK/ElvUI_Enhanced#140]
Random question.
I've seen on later versions of ElvUI that actionbar buttons can be modified to have different width/heights, to make rectangular buttons rather than the fixed squares.
Is there any chance of that appearing in this version? Or is WotLK too basic and hardcoded for it to work properly?

I could create weakauras for my entire actionbar, but that'll suck for vehicle combat/not include shift macro changing/and all sorts of reasons weakauras are difficult. Also sorry i posted this originally in the wrong place and can't delete it. My bad I suck at github.

**Comment by Exeasd** [comment]
You can just use Masque or ButtonFacade to reskin the bars hover you like

**Comment by Zelfie** [comment]
True, but none of the Masque/ButtonFacades I've seen are quite right.
I'd like to only have a certain 3x3 bar rectangluar to take up less space referencing my MMO mouse buttons, but my normal keys square.
It's available in later versions of ElvUI and would love the customisation is all.
